src/_guides/environment-declarations.md

```diff
@@ -0,0 +1,188 @@
+  // Log the debug message if the environment declaration 'DEBUG' is `true`.
+  // If there was no value specified, do not log.
+  if (const bool.fromEnvironment('DEBUG', defaultValue: false)) {
+    print('Debug: $message');
+  }
+}
+```
+
+In this snippet, if `DEBUG` is set to `false`
+during compilation, or not specified at all,
+production compilers can completely remove the condition and its body.
+
+The `fromEnvironment` constructors always
+have a default value for when the declaration isn't specified
+or the specified value cannot be parsed.
+Therefore, if you need to specifically check whether
+an environment declaration has been specified,
+use the [`bool.hasEnvironment`][bool-has] constructor:
+
+<?code-excerpt "misc/lib/development/environment_declarations.dart (has-debug)"?>
+```dart
+if (const bool.hasEnvironment('DEBUG')) {
+  print('Debug behavior was configured!');
+}
+```
+
+[string-from]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/String/String.fromEnvironment.html
+[int-from]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/int/int.fromEnvironment.html
+[bool-from]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/bool/bool.fromEnvironment.html
+[bool-has]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/bool/bool.hasEnvironment.html
+
+## Specifying environment declarations
+
+{{site.alert.warning}}
+  Dart tools and compilers currently do not
+  consistently handle environment declarations
+  with comma-separated values.
+  For more details,
+  see [SDK issue 44995][].
+{{site.alert.end}}
+
+[SDK issue 44995]: https://github.com/dart-lang/sdk/issues/44995
+
+### Dart CLI
+
+Both `dart run` and the `dart compile` subcommands accept
+any number of the `-D` or `--define` options
+to specify environment declaration values.
+
+```terminal
+$ dart run --define=DEBUG=true -DFLAVOR=free main.dart
+$ dart compile exe --define=DEBUG=true -DFLAVOR=free main.dart
+$ dart compile js --define=DEBUG=true -DFLAVOR=free main.dart
+$ dart compile aot-snapshot --define=DEBUG=true -DFLAVOR=free main.dart
+$ dart compile jit-snapshot --define=DEBUG=true -DFLAVOR=free main.dart
+$ dart compile kernel --define=DEBUG=true -DFLAVOR=free main.dart
+```
+
+#### `webdev`
+
+To learn about configuring `webdev` to pass environment declarations
+to both the development and production web compilers,
+see [the `webdev` configuration documentation][webdev-config].
+
+[webdev-config]: {{site.pub-pkg}}/build_web_compilers#configuring--d-environment-variables
+
+### Visual Studio Code
+
+In your launch configuration (`launch.json`) under `configurations`,
+add a new `toolArgs` key containing the your desired environment declarations:
+
+```json
+"configurations": [
+    {
+        "name": "Dart",
+        "request": "launch",
+        "type": "dart",
+        "toolArgs": [
+          "--define=DEBUG=true"
+        ]
+    }
+]
+```
+
+For more information, see the documentation for
```

Comment:
**chore:** Change `see` to `check out`. It's more inclusive language. Also, replace `For more information` to `To learn more`.